js/script.js

```diff
@@ -1,16 +1,70 @@
 $(function(){
-	var $carouselList = $('#carousel ul');
+	var $carouselList = $('#carousel ul'),
+		i = 0,
+		diff = 0,
+		diffNegative = 0,
+		holder,
+		photos = ['#one', '#two', '#three', '#four', '#five'],
+		interval = setInterval(changeSlideLeft, 3000);
+
 	$carouselList.css({marginLeft: -400});
-	var i = 0;
-	var photos = ['#one', '#two', '#three', '#four', '#five'];
+	$(photos[i]).removeClass('fa-circle-thin').addClass('fa-circle');
+	$('#right').click(function() {
+		changeSlideLeft();
+		resetInterval();
+	});
+	$('#left').click(function() {
+		changeSlideRight();
+		resetInterval();
+	});
 
-	photos.forEach(function(photo, index) {
+	$(document).keydown(function(e) {
+		if (e.keyCode === 39) {
+			changeSlideLeft();
+			resetInterval();
+		} else if (e.keyCode === 37) {
+			changeSlideRight();
+			resetInterval();
+		};
+    });
+
+    photos.forEach(function(photo, index) {
```

Comment:
Nie podoba mi się to co dzieje się w tej funkcji. Nie jest w żaden sposób odporna na dodawanie kolejnych zdjęć. Dodatkowo zbędne console.log